radar/tools/pm2.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "PM2"
+ring: adopt
+quadrant: "tools"
+tags: [nodejs, compute]
+---
+
+[PM2](https://pm2.keymetrics.io/) is a production process manager for Node.js applications with a built-in load balancer. It helps keep applications alive forever, reload them without downtime, and facilitate common system admin tasks.
+
+## Use cases
```

Comment:
Shall we add any reference to example usage?

Reply:
like?

Reply:
I'd pick some lines from this PR
https://github.com/pagopa/io-infra/pull/1190/files

Reply:
done